linux_os/guide/system/logging/journald/file_groupowner_journalctl/rule.yml

```diff
@@ -0,0 +1,37 @@
+documentation_complete: true
+
+
+title: 'Verify Groupowner on the journalctl command'
+
+description: |-
+    Verify that the "journalctl" command is group-owned by "root" by
+    using the following command:
+    <pre>
+    $ sudo find /usr/bin/journalctl -exec stat -c "%n %G" {} \;
```

Comment:
Wouldn't be simpler to use `ls -l`?

Reply:
It definitely would be :)
This is verbatim from the STIG guide. Should I simplify?

Reply:
Ah ok. I don't have strong opinion on this, but in general simpler is better.
I am fine with this if is aligned to the STIG guide. : )